jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/MultipleInputsTests.java

```diff
@@ -0,0 +1,260 @@
+
+        String surname = multipleInputs.getValue("#mi-i-2");
+        assertEquals(surname, inputData.get(2));
+
+        String value = multipleInputs.getValue();
+        assertEquals(value, inputData.get(1));
+    }
+
+    @Test
+    public void getValueAllTest() {
+        assertEquals(multipleInputs.getAllValues(), inputDataList);
+    }
+
+    @Test
+    public void setValueTest() {
+        multipleInputs.clearAll();
+
+        String value = inputData.get(1);
+        multipleInputs.setValue(value);
+        assertEquals(multipleInputs.getValue(1), value);
+
+        int index = 2;
+        String name = inputData.get(index);
+        multipleInputs.setValue(name, index);
+        assertEquals(multipleInputs.getValue(index), name);
+
+        String locator = "#mi-i-2";
+        String surname = inputData.get(2);
+        multipleInputs.clear(locator);
+        multipleInputs.setValue(surname, locator);
+        assertEquals(multipleInputs.getValue(locator), surname);
+    }
+
+    @Test
+    public void setAllValuesTest() {
+        multipleInputs.clearAll();
+        multipleInputs.setAllValues(inputDataList);
+        assertEquals(multipleInputs.getAllValues(), inputDataList);
+    }
+
+    @Test
+    public void sendKeysTest() {
+        multipleInputs.clearAll();
+
+        String value = inputData.get(1);
+        multipleInputs.sendKeys(value);
+        assertEquals(multipleInputs.getText(), value);
+
+        multipleInputs.clearAll();
+        int index = 2;
+        String name = inputData.get(index);
+        multipleInputs.sendKeys(index, name);
+        assertEquals(multipleInputs.getText(index), name);
+
+        multipleInputs.clearAll();
+        String locator = "#mi-i-2";
+        String surname = inputData.get(2);
+        multipleInputs.sendKeys(locator, surname);
+        assertEquals(multipleInputs.getText(locator), surname);
+    }
+
+    @Test
+    public void sendAllKeysTest() {
+        multipleInputs.clearAll();
+
+        multipleInputs.sendKeysAll(inputDataList);
+        assertEquals(multipleInputs.getAllTexts(), inputDataList);
+    }
+
+    @Test
+    public void clearTest() {
+        String value = inputData.get(1);
+        multipleInputs.sendKeys(value);
+        multipleInputs.clear();
+        assertEquals(multipleInputs.getText(), "");
+
+        int index = 2;
+        String name = inputData.get(index);
+        multipleInputs.sendKeys(index, name);
+        multipleInputs.clear(index);
+        assertEquals(multipleInputs.getText(index), "");
+
+        String locator = "#mi-i-2";
+        String surname = inputData.get(2);
+        multipleInputs.sendKeys(locator, surname);
+        multipleInputs.clear(locator);
+        assertEquals(multipleInputs.getText(locator), "");
+    }
+
+    @Test
+    public void clearAllTest() {
+        multipleInputs.sendKeysAll(inputDataList);
+        multipleInputs.clearAll();
+
+        multipleInputs.getAllTexts().forEach(t -> assertEquals(t, ""));
```

Comment:
it's better to use jdi assert instead, refactor to something like that: multipleInputs.getInputs().forEach(input->input.is().text(""));